README.md

```diff
@@ -79,42 +77,50 @@ In Wave 1 we will create the `Vendor` class.
 
 ### Wave 2
 
-In Wave 2 we will create the `Item` class and the `get_by_category` method.
+In Wave 2 we will create the `Item` class and the `Vendor` class' `get_by_id` method.
 
 - There is a module (file) named `item.py` inside of the `swap_meet` package (folder)
 
 - Inside this module, there is a class named `Item`
-- Each `Item` will have an attribute named `category`, which is an empty string by default
-- When we initialize an instance of `Item`, we can optionally pass in a string with the keyword argument `category`
-- Instances of `Vendor` have an instance method named `get_by_category`
-  - It takes one argument: a string, representing a category
-  - This method returns a list of `Item`s in the inventory with that category
+- Each `Item` will have an attribute named `id`, which is a unique integer by default
+  - There are many ways to generate numbers, but generating numbers without duplicates takes some care. Happily, Python has a package called `uuid` that can help!
+    - If we import the [`uuid` package](https://docs.python.org/3/library/uuid.html) in `item.py`, with a little research we can use one of the functions `uuid` provides to create large unique numbers meant to be used as identifiers
+    - This package creates `UUID` objects, its functions don't directly return an integer, **but** `UUID` instances have [an attribute `int`](https://docs.python.org/3/library/uuid.html#uuid.UUID.int) which allow us to access their value as an integer
+- When we initialize an instance of `Item`, we can optionally pass in an integer with the keyword argument `id` to manually set the `Item`'s `id`
+- Each `Item` will have a function named `get_category`, which will return a string holding the name of the class
+
+- Instances of `Vendor` have an instance method named `get_by_id`
+  - This method takes one argument: an integer, representing an `Item`'s `id`
+  - This method returns the item with a matching `id` from the inventory
+  - If there is no matching item in the `inventory`, the method should explicitly return `None`
 
 ### Wave 3
 
-In Wave 3 we will write a method to stringify an `Item` using `str()` and write the method `swap_items`.
+In Wave 3 we will write a method to stringify (convert to a string) an `Item` using `str()` and write the method `swap_items`.
 
-- When we stringify (convert to a string) an instance of `Item` using `str()`, it returns `"Hello World!"`
+- When we stringify an instance of `Item` using `str()`, it returns `"An object of type Item with id <id value>"`, where `<id value>` is the `id` of the `Item` instance that `str()` was called on.
+  - For example, if we had an `Item` instance `item_a = Item(id=12345)`, the output of `str(item_a)` should be `"An object of type Item with id 12345"`.
   - This implies `Item` overrides its stringify method. We may need to research the `__str__` method for more details!
```

Comment:
I'm wondering if instead of asking folks to research the `__str__` method, we instead add a section at the bottom of the README with a brief explanation of operator overloading and what we might expect `str(item_instance)` to return if we didn't write our own `str()` method. Or we could even include the explanation directly in the bullet points here.

I see a lot of folks in internship who don't seem to fully understand the concept.